src/ol_infrastructure/applications/ocw_build/__main__.py

```diff
@@ -95,6 +95,7 @@
         }
     ]
 }
+lint_iam_policy(ocw_next_instance_policy_data)
```

Comment:
Not related specifically to this PR, but do we want to standardize on how we refer to app policies? Is it `<app_name>_policy` or `<app_name>_policy_data` or something else ...